### DISCUSSION TITLE
No Contract Compiled Yet

### DISCUSSION THREAD

**Discussion by mSalimrKhan** [smartcontractkit/full-blockchain-solidity-course-py#629] [Q&A] [answered]
When compile my problem facing can u guide what is problem in this code
![image](https://user-images.githubusercontent.com/93871219/147386407-e639fcd4-3fba-4202-904b-9683e4ad991f.png)
facing

**Comment by cromewar** [comment, marked as answer]
Hello @mSalimrKhan on line 6 is:
`uint256 _favoriteNumber;` and you have `unite256 _favoriteNumber`
Give it a try, best regards Cromewar.

**Reply by mSalimrKhan** [reply]
@cromewar   for reply me can u give any project i want work on that and also learn a lot from it

**Reply by mSalimrKhan** [reply]
Thnx a lot to @cromewar

**Reply by cromewar** [reply]
If you want project ideas you can check this video by moralis: https://www.youtube.com/watch?v=NuV13bh2PMo
I'm glad the problem is solved now I'm going to close this thread now.
